README.md

```diff
@@ -1886,28 +1886,34 @@ $ git signatures show v1.0.0
  ```
 
 ## [git-profile](https://github.com/dm3ch/git-profile-manager)
-
+### add a work profile
 ```
-# add a work profile
 $ git profile add work
 Name: Name Surname
 Email: name@work-domain.com
 Signing Key:
 Profile work added successfully
+```
 
-# add a personal profile
+### add a personal profile
+```
 $ git profile add home -n "Name Surname" -e name@gmail.com
 Profile home added successfully
+```
 
-# list out our saved profiles
+### list out our saved profiles
+```
 $ git profile list
 Existing profiles:
 work
 home
+```
 
-# set the current git repository user to the home profile
+### set the current git repository user to the home profile
+```
 $ git profile use work
-# Currently there will be no output in case of success
+```
+Currently there will be no output in case of success
```

Comment:
Would be great if there's a success message printed :)

Reply:
Yeah. I know, I already created issue to make all commands print some successful message before exit.